server/src/routers/api/userRouter.js

```diff
@@ -33,6 +33,15 @@ router.post("/login" , async(req,res)=>{
   }
 })
 
+router.get("/profile", async(req,res)=> {
+  try{
```

Comment:
You have to send complete user profile ( refer profile model )
Also, you have to populate the userField.

Reply:
@Codess08  The purchases and doctors models are not yet created. Without them, they can not be populated.

Reply:
Okay, so you can take up other issues for now.
I won't unassign this issue don't worry.

Reply:
Okay, thanks